.github/workflows/pr.yaml

```
@@ -60,30 +98,38 @@ jobs:
 
   test:
     name: Test with Node.js ${{ matrix.node-version }}
+    needs: check-image
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [22]
+        node-version: [ 20, 22 ]
```

Why are we re-adding node 20?

```suggestion
        node-version: [ 22 ]
```